docs/syntax.rst

```diff
@@ -29,102 +25,71 @@ The return value will be automatically casted into the hinted type. e.g.,
     res = add_xy(2.3, 1.1)
     print(res)  # 3, since return type is ti.i32
 
-.. note::
-    For differentiable programming kernels should better have either serial statements or a single parallel for-loop. If you don't use differentiable programming, feel free to ignore this tip.
 
 .. note::
 
-    For now, we only support one scalar as return value. Returning ``ti.Matrix`` or `ti.Vector`` is not supported. Python-style tuple return is not supported. e.g.:
+    For now, we only support one scalar as return value. Returning ``ti.Matrix`` or ``ti.Vector`` is not supported. Python-style tuple return is not supported either. For example,
```

Comment:
```suggestion
    For now, we only support one scalar as return value. Returning ``ti.Matrix`` or ``ti.Vector`` is not supported. Python-style tuple return is not supported either. e.g.:
```

Reply:
Why is `e.g.:` better? In English people usually use `E.g.,`, which is the same as `For example,`.

Reply:
```suggestion
    For now, we only support one scalar as return value. Returning ``ti.Matrix`` or ``ti.Vector`` is not supported. Python-style tuple return is not supported either. For example:
```
Fine, but I found that `poedit` hate this, when en ends with `,`, and cn ends with `：`, it WARNs you incorrect, and you can't get rid that WARN... Let's use `:` for `e.g.`'s, which helps both en and cn, wdyt?

Reply:
Thanks. Sounds good to me. No perfect solution here I guess.